SPECS/openvswitch/openvswitch.spec

```diff
@@ -201,6 +201,7 @@ make -k check |& tee %{_specdir}/%{name}-check-log || %{nocheck}
 %{_unitdir}/ovs-vswitchd.service
 %{_unitdir}/ovsdb-server.service
 %{_libdir}/lib*
+%exclude %{_libdir}/*.a
```

Comment:
Having to parse `%exclude` makes it a bit harder to understand what the package produces. I'd suggest removing this line and using `%{_libdir}/lib/*so*` in the line above instead. From what I can see this should cover what you're doing now. Extra points for splitting out each shared library separately so that their major version is visible. Example:

```
%{_libdir}/lib/libofproto-2.12.so.0*
%{_libdir}/lib/libofproto.so
%{_libdir}/lib/libopenvswitch-2.12.so.0*
%{_libdir}/lib/libopenvswitch.so
(...)
```